app.js

```diff
@@ -41,12 +44,26 @@ app.use(cors());
 app.use(logger('dev'));
 app.use(express.json());
 app.use(express.urlencoded({ extended: false }));
-app.use(cookieParser());
+app.use(cookieParser()); // DO NOT MODIFY THIS LINE, IT MAY SCREW UP EXPRESS-SESSION
+if (env == 'production' || env == 'test') {
+    var RedisStore = require('connect-redis')(session);
+    var redisClient = redis.createClient();
+}
+app.use(
+    session({
+        store:
```

Comment:
the meaning of setting an expiration date is that after a while the user will suddenly be disconnected. it doesn't make since. I find myself connected to websites I havn't log into in years.
I'v added the secure cookie